Classification_Algorithm.py

```diff
@@ -0,0 +1,140 @@
+# Data Preprocessing
+import pandas as pd
+import matplotlib.pyplot as plt
+from sklearn.preprocessing import StandardScaler
+from sklearn.cluster import KMeans, Birch
+from sklearn_extra.cluster import KMedoids
+from sklearn.metrics import silhouette_score, calinski_harabasz_score
+from warnings import simplefilter
+
+# ignore all future warnings
+simplefilter(action='ignore', category=FutureWarning)
+
+df = pd.read_csv('Mall_Customers_Preprocessed.csv')
+df.dropna(inplace=True)
+
+# Select relevant features and scale data
+X = df.iloc[:, [2,3]].values
```

Comment:
Let's cluster four dimensions even if we only display the results in a two or three dimensional scatter plot. :)

Reply:
Yeah i tried doing that. Our Gender column with binary values kinda disturbs the clustering. 
Example of how it looks when we use all columns.
![image](https://github.com/WinstonWaligora/SE_412_512_Datamining_Group_Project/assets/89600582/a8db6489-09f9-4292-8dba-1a5dc84d4510)

Reply:
It makes sense now why they chose relevant features instead of all features.